docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -763,58 +763,57 @@ file a ticket or issues. That's a bug.
 
 Every task has some expansions available by default:
 
--   `${is_patch}` is "true" if the running task is in a patch build and
-    undefined if it is not.
--   `${is_stepback}` is "true" if the running task was stepped back.
+
 -   `${author}` is the patch author's username for patch tasks or the
     git commit author for git tasks
 -   `${author_email}` is the patch or the git commit authors email
--   `${task_id}` is the task's unique id
--   `${task_name}` is the name of the task
--   `${execution}` is the execution number of the task (how many times
-    it has been reset)
 -   `${build_id}` is the id of the build the task belongs to
--   `${build_variant}` is the name of the build variant the task belongs
-    to
--   `${version_id}` is the id of the task's version
--   `${workdir}` is the task's working directory
--   `${revision}` is the commit hash of the base commit that a patch's changes
-    are being applied to, or of the commit for a mainline build. For PR patches,
-    this is the merge base of the PR branch and the target branch.
--   `${github_commit}` is the commit hash of the commit that triggered
-    the patch run
 -   `${branch_name}` is the name of the branch tracked by the
     project
--   `${distro_id}` is name of the distro the task is running on
+-   `${build_variant}` is the name of the build variant the task belongs
+    to
 -   `${created_at}` is the time the version was created
--   `${revision_order_id}` is Evergreen's internal revision order
-    number, which increments on each commit, and includes the patch
-    author name in patches
--   `${github_pr_number}` is the Github PR number associated with PR
-    patches and PR triggered merge queue items
--   `${github_org}` is the GitHub organization for the repo in which
-    a PR or PR triggered merge queue item appears
--   `${github_repo}` is the GitHub repo in which a PR or PR triggered
-    merge queue item appears
+-   `${distro_id}` is name of the distro the task is running on
+-   `${execution}` is the execution number of the task (how many times
+    it has been reset)
 -   `${github_author}` is the GitHub username of the creator of a PR
     or PR triggered merge queue item
+-   `${github_commit}` is the commit hash of the commit that triggered
+    the patch run
 -   `${github_known_hosts}` is GitHub's SSH key fingerprint
--   `${triggered_by_git_tag}` is the name of the tag that triggered this
-    version, if applicable
+-   `${github_org}` is the GitHub organization for the repo for the project
+-   `${github_pr_number}` is the Github PR number associated with PR
+    patches and PR triggered merge queue items
+-   `${github_repo}` is the GitHub repo for the project
```

Comment:
I like that this is alphabetized! However, this is my personal preference so no worries if you disagree, but can we group `github_org` and `github_repo` together? I am used to thinking of them as a unit: owner/repo. 
I also have a slight preference for using `github_owner` instead of `github_org`  here.

Reply:
i know right haha i couldn't decide, I will group them. I'm going to keep this as github_org though, since this isn't a new variable, to keep this backwards compatible (personally though I do prefer org since that's what users more understand and correlates better with GitHub documentation; "owner" is more just something we use in our codebase).